distributions/distributions.json

```diff
@@ -0,0 +1,12 @@
+{
+    "LastCumulativeAmount": "340723072250877984478741713",
```

Comment:
Sorry one nit-pick, and I know I provided this name, could we change this to "LatestCumulativeAmount"?

Reply:
Absolutely. I like it. Done